projects/packages/stats/src/class-tracking-pixel.php

```diff
@@ -85,6 +87,41 @@ public static function build_view_data() {
 			}
 		}
 
+		if ( $is_archive ) {
+			if ( $wp_the_query->is_home() ) {
+				$view_data['home'] = '1';
+			} elseif ( $wp_the_query->is_archive() ) {
+				if ( $wp_the_query->is_date ) {
```

Comment:
Tricky corner case... these taxonomies are not actually completely exclusive from each other. You can specify a combination of different ones together in the url as paths and as query params.

Example: https://datalinkp2.wordpress.com/author/gibrown/?tag=team-update which is also the same as https://datalinkp2.wordpress.com/?author=gibrown&tag=team-update

Reply:
I think it is better to parse them in the scope of the tracking code. Should these be handled as a combination in the backend though, or is it enough to handle them as separate logs?

Reply:
I've updated the code once more to track down multiple types of archive pages. As I mentioned above, I think it's better to do it in the scope of tracking data. @gibrown, let me know if everything looks good, or else I can revert my last changes.

Reply:
While the proposed approach ensures we don't override previous `arch` entries when looping over the archive parameters, I'm a bit concerned about how we will handle this in the backend. For example, if a log entry contains both `arch_tag` and `arch_author`, as the mentioned example, will it count as a view increment for both separately, resulting in two views instead of one?

Reply:
> For example, if a log entry contains both `arch_tag` and `arch_author`, as the mentioned example, will it count as a view increment for both separately, resulting in two views instead of one?

I had the same thoughts in my comment above. I believe that providing stats of combinations of such types won't provide much value at this point, but I could be wrong. All things considered, we can start simple in the first iteration and improve by providing such stats for combinations of archives if needed in the future.

Reply:
This all looks good. And ya I think we can play a bit with the backend about how we want to store and track them all.

> I believe that providing stats of combinations of such types won't provide much value at this point, but I could be wrong

I think maybe our designs should lean towards having a category that is "Multiple Filters" or something like that. So we can then just group those cases all together.